config/prod/nkauer-data-curation.yaml

```diff
@@ -0,0 +1,48 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-linux-v1.j2
+stack_name: nkauer-r-studio
```

Comment:
please name it the same as the file name or vice versa

Reply:
Whoops. Renamed.